--- a/misterynano_tc138k/tang/console138k/top.sv
+++ b/misterynano_tc138k/tang/console138k/top.sv
@@ -240,8 +240,6 @@ assign i2s_lrck = por?1'b0:audio_bit_cnt[4];
 assign i2s_din = por?1'b0:audio[i2s_lrck][15-audio_bit_cnt[3:0]];
    
 misterynano misterynano (
-  .clk   ( clk ),           // 50MHz clock uses e.g. for the flash pll
-
   .reset ( 1'b0), // !reset_n ), disabled to fix tc138k booting, needs to be investigated !!!
   .user  ( 1'b0), // !user_n ),
